### ISSUE TITLE
Store And Use SRS in the QRiS project

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/QRiS#225]
QRiS stores data in WGS84, but people nearly always want to work in a cartesian coordinate system.

That said, we don't want to roll our own SRS user interface.... it's extremely complicated!!!!!

My proposal:

1. Let the user specify the map document SRS using the built in QRiS user interface. 
2. On project close we will read the current map document SRS and store it in the metadata column of the projects table in the QRiS GeoPackage.
3. We will also add a "Save Project SRS" right click menu item on the project node of the project tree (this gives us a sure fire way of making sure it saves in case of a QGIS crash. Easier to test too!)
4. On loading a QRiS project we will look for this SRS and use it to set the current map document to this SRS if it is not already. If there's no SRS stored then do nothing.

In summary... we will continue to store data in QGS84. But we want users to view data in projected coordinates so they can measure things etc.

**Comment by KellyMWhitehead** [comment]
Needs further investigation. Storing and retrieving the SRS is easy, but managing the map SRS is tricky when there are no layers in the map (QGIS overrides these settings).

Parking in backlog for now.

**Comment by joewheaton** [comment]
Lets revisit if this is still relevant after 1.0 @philipbaileynar and @KellyMWhitehead

**Comment by KellyMWhitehead** [comment]
This is fixed for the upcoming 1.3 release.

**Comment by philipbaileynar** [comment]
See #1096
